File: app/src/main/java/com/przemyslawlusnia/vocabularycreator/wordlist/presentation/WordsView.java
package com.przemyslawlusnia.vocabularycreator.wordlist.presentation;

import com.przemyslawlusnia.vocabularycreator.core.BaseView;
import java.util.List;

public interface WordsView extends BaseView {

  void showAddWord(WordViewModel wordViewModel);

  void showAllWordsAfterDeletion();

  void showAllWords(List<WordViewModel> wordViewModels);
}

File: app/src/main/java/com/przemyslawlusnia/vocabularycreator/wordlist/presentation/WordsView.kt
package com.przemyslawlusnia.vocabularycreator.wordlist.presentation

import com.przemyslawlusnia.vocabularycreator.core.BaseView

interface WordsView : BaseView {

    fun showAddWord(wordViewModel: WordViewModel)

    fun showAllWordsAfterDeletion()

    fun showAllWords(wordViewModels: List<WordViewModel>)
}
